plotRisk.py

```diff
@@ -74,4 +80,4 @@ def heal(): # searches food items for needed nutritional value then consumes the
 char.magicDefence = 100
 char.defence = 100
 char.health = 10
-char.plotRisk(att)
\ No newline at end of file
+char.plotRisk(att)
```

Comment:
> AttributeError

Reply:
I dunno python, not sure what is wrong here

Reply:
It's because it should be `PlotRisk` not `plotRisk`

Reply:
I fixed it with the latest commit

Reply:
Did you push that commit?

Reply:
![image](https://github.com/chanson02/SENG3340/assets/119390131/3f2a5f3a-ed8e-4057-a6b1-16e66930bbb8)

Reply:
That commit fixed the issue on line 77, not this one (line 83).

Reply:
Good catch, should be fixed now